### DISCUSSION TITLE
How to use the " Lovelace View" option in the doorbell notification blueprint?

### DISCUSSION THREAD

**Discussion by itkama** [briis/unifiprotect#435] [Q&A] [answered]
Hi there,

thanks for this awesome integration! I've just setup the doorbell notification blueprint.
I would like to be able to open a specific lovelace dashboard and there seems to be an option for that:
![image](https://user-images.githubusercontent.com/37951363/147695722-c08b1ac9-f450-44f9-b9e9-75319d74395f.png)
But in what format does this textfield want the information?
I've got the dashboard in question under this link: `https://ha.mydomain.xyz/specificDashboard/0`
Which part of that do I need to use? The whole link or just "specificDashboard" didn't do the trick.

Best Regards

Manuel

**Comment by AngellusMortis** [comment, marked as answer]
Just the URI path. So `/specificDashboard/0`. On Android/iOS it will navigate to that path within the app. 

If you are using HTML 5 push notifications, you also need to make sure "Base Home Assistant URL" is filled out. It will open base URL + Lovelace view in your default browser when you click on it.

Telegram notifications do not support opening Lovelace views.

**Comment by itkama** [comment]
So I get a button to "open camera" now after filling out the "Lovelace View" option correctly:
![image](https://user-images.githubusercontent.com/37951363/147853032-773eb7d4-0ceb-4754-810a-f14666b7630f.png)
But if I understand the description correctly, I should also get here, if I just tap on the image? Because if I do that, I just get to my default lovelace view in the App. The button "open camera" takes me to my desired destination though.
Is this the right behaviour or am I still doing something wrong?

**Reply by AngellusMortis** [reply]
If you are using Android/iOS, then yes you get an action that takes you to the view ("Open Camera") if you are using HTML 5 push, then it is just clicking the notification itself opens it.
